### DISCUSSION TITLE
Why are they all quoted from.Lowdefy files? Hurry

### DISCUSSION THREAD

**Discussion by xwen107** [lowdefy/lowdefy#1615] [Questions and Help]
![Uploading image.png…]()

**Comment by xwen107** [comment]
![image](https://github.com/lowdefy/lowdefy/assets/134669335/6575dd60-9145-41c7-a062-94f5106d4a5b)

**Reply by xwen107** [reply]
Why are they all quoted from.Lowdefy files? Hurry

**Reply by SamTolmay** [reply]
Hi @xwen107 

Are you using a `resolver` or `transformer` function?

**Reply by xwen107** [reply]
Yes it is

**Reply by xwen107** [reply]
So what am I supposed to do @SamTolmay

**Reply by Gervwyk** [reply]
You need to provide more details here. Please post what `appFramework/templates/lowdefy.yaml.njk` looks like?

**Reply by xwen107** [reply]
![9756f4def2b72864d96c87a365aaa6a](https://github.com/lowdefy/lowdefy/assets/134669335/0e9f6059-a885-4c1e-bd7f-3a11f075f681)

**Reply by xwen107** [reply]
Is this what you want？

**Reply by Gervwyk** [reply]
Yes, why do you have the `refResolver` specified? Are you using it? 
Also a bigger issue it `_ref` operator, like all operators must to be the only key in its object. Should it not be under `pages`?
